bindings/nodejs/src/wallet.rs

```diff
@@ -99,7 +99,7 @@ pub fn create_wallet(mut cx: FunctionContext) -> JsResult<JsBox<WalletMethodHand
 pub fn call_wallet_method(mut cx: FunctionContext) -> JsResult<JsUndefined> {
     let method = cx.argument::<JsString>(0)?;
     let method = method.value(&mut cx);
-    let method_handler = Arc::clone(&&cx.argument::<JsBox<WalletMethodHandlerWrapper>>(1)?.0);
```

Comment:
Are you using different clippy rules? Or why wasn't this detected earlier ?

Reply:
There was an allow for `needless_borrow`